src/Toggle/ToggleChecker.php

```diff
@@ -51,9 +46,28 @@ public function isToggleActive( $toggleName )
         if ( !$this->toggleGateway->isToggleVisibleForUsers($toggleName) ) {
             return false;
         }
-        $groupPolicyResponse = $this->groupPolicy->getTogglePolicy($toggleName, $this->group);
+        $groupPolicyResponse = $this->transformByType(
+            $this->groupPolicy->getTogglePolicy($toggleName, $this->group),
+            $this->toggleGateway->getType($toggleName));
         return $groupPolicyResponse->isEnabled() ||
         ( $groupPolicyResponse->isNotSet() && $this->userPolicy->getTogglePolicy($toggleName, $this->user)->isEnabled());
     }
+
+    /**
+     * @param UseCase\Response\TogglePolicyResponse $groupPolicyResponse
+     * @param string $type
+     * @return TogglePolicyResponse
+     */
+    private function transformByType(UseCase\Response\TogglePolicyResponse $groupPolicyResponse, $type)
+    {
+        switch ($type) {
+            case self::TOGGLE_TYPE_GROUP:
+                if ( $groupPolicyResponse->isNotSet() ) {
+                    return new TogglePolicyResponse(true,false);
```

Comment:
Creating a different TogglePolicyResponse object feels a little bit weird here. Though it's only for internal usage and not visible from outside, so I'm not sure.

Also this constructor is a little bit strange as well, e.g. passing ( false, true ) is kind of pointless. In my implementation I used only one argument (maybe the name is not the best), and I check for 3 values of the variable, true, false and null:

https://github.com/clearbooks/labs-php-client-mysql/blob/is-toggle-active/src/Toggle/TogglePolicyResponse.php